docs/self-managed/operational-guides/update-guide/870-to-880.md

```diff
@@ -6,6 +6,140 @@ description: "Review which adjustments must be made to migrate from Camunda 8.7.
+
+### Notes for updating to 8.8
+
+When updating from Camunda 8.7 to 8.8, the following should be considered regarding the Camunda Exporter and harmonized indices.
+
+#### Ensure the exporter/import backlog is small
+
+For data from 8.7 to be imported, the importers must run first and complete before the Camunda Exporter can start.
+
+The importer reads data from existing Zeebe Elasticsearch indices, and when it detects data from newer versions, it marks itself as complete. The Camunda Exporter then starts exporting only when the importer is done.
+
+:::important
+To reduce the potential of building a backlog,
+the user needs to make sure to drain the backlog queue as much as possible before the update.
+:::
+
+The backlog can be validated via the following Prometheus metrics:
+
+- `operate_import_time_seconds_sum`
+- `operate_import_time_seconds_count`
+
+For example, the following query shows the import latency. If the result is in a range of seconds, it is fine to update.
+
+```promql
+sum(rate(operate_import_time_seconds_sum{namespace=~"$namespace", partition=~"$partition", pod=~"$pod"}[$__rate_interval]))
+/ sum(rate(operate_import_time_seconds_count{namespace=~"$namespace", partition=~"$partition", pod=~"$pod"}[$__rate_interval]))
+```
+
+#### Data migration
+
+To enable Tasklist to utilize the new harmonized indices, Camunda 8.8 provides a migration application to enhance existing process models. This migration application runs asynchronously to the other applications until the importers are done, and ensures that all processes created in 8.7 are migrated and enhanced.
+
+The migration application can be found in the `camunda/bin/migrate` directory.
+
+:::note
+When using the Camunda 8 Helm charts, the migration application will automatically deploy with the importers.
+:::
+
+#### Turn off importers after completion
+
+Importers are only necessary at the start of the migration. Once complete, they can be safely turned off.
+
+To detect whether importers are done, users can validate the state in the Elasticsearch/OpenSearch index:
+
+- `tasklist-import-position-8.2.0_`
+- `operate-import-position-8.3.0_`
+
+If all entries show the `completed` property set to `true`, the importing is done.
+
+Alternatively, metrics can be validated via:
+
+- `operate_import_completed`
+- `tasklist_import_completed`
+
+These properties can be used in the following example query:
+
+```
+sum(tasklist_import_completed{namespace="$namespace"}) by (importPositionAlias)
+```
+
+![metrics-import-complete](./img/import-complete-metrics.png)
+
+#### Update any custom prefixes
+
+:::important
+This step is required if you use an installation with custom prefixes.
+:::
+
+The new harmonized index schema introduces one common index prefix. To update to Camunda 8.8, indices must first migrate to one common index, so the Camunda Exporter, REST API, and other consumers can be configured correctly.
```

Comment:
I couldn't quite tell in this second sentence if "indices must first migrate to one common index" meant "existing indices using custom prefixes must be migrated to a common prefix" - is something along those lines more descriptive? It would be nice to be clear.

Reply:
Makes sense, or sounds right.